update information-schema/information-schema-tiflash-replica.md (#10458)
pingcap/docs: quick-start-with-htap.md

In the result of the above statements:

> - `AVAILABLE` indicates whether the TiFlash replica of a specific table is available or not. `1` means available and `0` means unavailable. Once a replica becomes available, this status does not change any more.

Reviewer: Once `AVAILABLE` becomes `1`, this status does not change any more.

> - `AVAILABLE` indicates whether the TiFlash replica of a specific table is available or not. `1` means available and `0` means unavailable. Once a replica becomes available, this status does not change any more.

Author: ```suggestion
- `AVAILABLE` indicates whether the TiFlash replica of a specific table is available or not. `1` means available and `0` means unavailable. Once the `AVAILABLE` field becomes `1`, this status does not change anymore.
```
